email_mailbox/src/libs/signal.js

```diff
@@ -21,9 +24,13 @@ const decryptEmail = async ({
     return text;
   }
   const textEncrypted = util.toArrayBufferFromBase64(text);
+  const [recipientIdFrom, deviceIdFrom] =
+    external === true
```

Comment:
Check for external outside `signal.js` and pass the corresponding recipientId-deviceId pair to this method